modules/470-chrony/images/chrony-exporter/werf.inc.yaml

```diff
@@ -0,0 +1,46 @@
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-promu-artifact
```

Comment:
pls, modify build using `*-src-artifact` (similar to other builds)

Reply:
@KraMorK , can you help with build if will any questions?

Reply:
believe u mean something like i made in last commit, check pls

Reply:
it is wrong way. For example we need to do some like this https://github.com/deckhouse/deckhouse/pull/11062/files#diff-48dfbab5a55df0d0d0851f2d8abf7af48ba04d21c2dd0ae5e4e670a1d9639ac6